src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -85,8 +158,11 @@ export const StakingOpportunities = ({ assetId, opportunities }: StakingOpportun
     e.stopPropagation()
   }
 
-  const handleStakedClick = () => {
-    cosmosStaking.open({ assetId: 'cosmos:cosmoshub-4/slip44:118' })
+  const handleStakedClick = (values: Row<object>) => {
```

Comment:
[gomesalexandre](https://github.com/gomesalexandre) [yesterday](https://github.com/shapeshift/web/pull/1356/files#r839990832)
Don't use object, it's an old construct from TS that we shouldn't use.
We should be able to type things more accurately than any object.